index.html

```diff
@@ -0,0 +1,124 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Product Landing Page</title>
+    <link
+  rel="stylesheet"
+  href="https://use.fontawesome.com/releases/v5.8.1/css/all.css"
+  integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf"
+  crossorigin="anonymous"
+/>
```

Comment:
Kindly endeavour to maintain proper indentation in your code. This will help foster better readability in your code both to others and yourself when you revisit your code at a later time.
On VS Code, extensions like 'Prettier' could be of assistance.

Reply:
Good job 👍